curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f0a55847d6cc485f29ba5.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 660f0a55847d6cc485f29ba5
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+A <dfn>method</dfn> is a special function attached to certain values in JavaScript. You have already used a method without realising it - the `.log()` method of the `console` object.
```

Comment:
```suggestion
A <dfn>method</dfn> in JavaScript is a function that's associated with certain values or objects. An example you've already encountered is the `.log()` method, which is part of the `console` object.
```

Reply:
Math.random() for object
'random'.split('dom') for values

Reply:
Huh?